### ISSUE TITLE
Need a way to detect symbols in object literal keys and values

### ISSUE THREAD

**Issue by SkReD** [ziolko/intellij-javascript-symbols#7]
In my code i frequently use such constructions for define event listeners:
...

```
handlers:
{
    'OnSomeEvent': function(emitter)
    {
       [some code]
    }
}

```

And somewhere in the code:
`this.Trigger('OnSomeEvent');`

Second example 'OnSomeEvent' is detected as symbol, but first is not. So i get warning, that symbol not used.

**Comment by ziolko** [comment]
Hi @SkReD,

Thank you for this feature request. I've encountered this problem as well and I think it's a good idea to handle object property names.

I am not sure if I will be able to provide all functionalities(rename, find usages, go to declaration) for property keys but I am almost sure they will be recognized as symbol usages.

Hopefully I will release a version with this functionality included on weekend or during next week :)

Kind Regards,
Mateusz

**Comment by SkReD** [comment]
Thanks a lot @ziolko!

**Comment by ziolko** [comment]
Hi @SkReD,

I've just released a new version (2.1) with this functionality included. Now all property names which are string literals (it means all property names in single or double quote) are treated as symbols.

Please let me know if it works for you.

Have a nice day,
Mateusz

**Comment by SkReD** [comment]
Hi @ziolko,

Works perfect. Thanks for your work!
